x_estate/models/x_real_estate_property_offer_fields.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="field_real_estate_property_offer_price" model="ir.model.fields">
+        <field name="model_id" ref="real_estate_property_offer_model"/>
+        <field name="name">x_price</field>
+        <field name="field_description">Price</field>
+        <field name="ttype">float</field>
+        <field name="required">True</field>
+    </record>
+
+    <record id="field_real_estate_property_offer_status" model="ir.model.fields">
+        <field name="model_id" ref="real_estate_property_offer_model"/>
+        <field name="name">x_status</field>
+        <field name="field_description">Status</field>
+        <field name="ttype">selection</field>
+        <field name="selection">True</field>
+        <field name="selection">[('x_accepted', "Accepted"), ('x_refused', "Refused")]</field>
```

Comment:
it makes no sense to define twice the same field (only last one is taken into account)